ledger/ledger-api-test-tool/conformance.bzl

```diff
@@ -16,29 +17,31 @@ def conformance_test(
         ports = [6865],
         test_tool_args = [],
         tags = [],
-        client = "//ledger/ledger-api-test-tool",
         runner = "@//bazel_tools/client_server/runner_with_port_check:runner",
-        flaky = False):
-    client_server_test(
-        name = name,
-        runner = runner,
-        runner_args = ["%s" % port for port in ports],
-        timeout = "long",
-        client = client,
-        client_args = test_tool_args + ["localhost:%s" % port for port in ports],
-        data = extra_data,
-        server = server,
-        server_args = server_args,
-        tags = [
-            "dont-run-on-darwin",
-            "exclusive",
-        ] + tags,
-        flaky = flaky,
-    ) if not is_windows else None
+        flaky = False,
+        lf_versions = ["stable", "latest", "preview"]):
+    for lf_version in lf_versions_aggregate(lf_versions):
+        if not is_windows:
+            client_server_test(
+                name = "-".join([name, lf_version]),
+                runner = runner,
+                runner_args = ["%s" % port for port in ports],
+                timeout = "long",
+                client = "//ledger/ledger-api-test-tool:ledger-api-test-tool-%s" % lf_version,
+                client_args = test_tool_args + ["localhost:%s" % port for port in ports],
+                data = extra_data,
+                server = server,
+                server_args = server_args,
+                tags = [
+                    "dont-run-on-darwin",
+                    "exclusive",
+                ] + tags,
+                flaky = flaky,
+            )
 
-def server_conformance_test(name, servers, server_args = [], test_tool_args = [], flaky = False):
+def server_conformance_test(name, servers, server_args = [], test_tool_args = [], flaky = False, lf_versions = ["stable", "latest", "preview"]):
```

Comment:
Should `lf_versions` here be passed down to the call to the `conformance_test` rule?